package.json

```diff
@@ -98,7 +98,7 @@
     "codemirror": "5.44.0"
   },
   "optionalDependencies": {
-    "wescheme-js": "bootstrapworld/wescheme-js"
+    "wescheme-js": "github:bootstrapworld/wescheme-js"
```

Comment:
Not sure if we want to include this change...

Reply:
What's the benefit of this change?

Reply:
It happened auto-magically after I installed `wescheme-js`.

Reply:
Huh. OK. Well, I'm good to merge this. Nice work!